config/settings.py

```diff
@@ -32,7 +31,7 @@
 SECRET_KEY = env('SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = env('DEBUG')
+DEBUG = False
```

Comment:
This should remain tied to the environment variable - could you revert this change or maybe include a default if the environment variable doesn't exist?

Reply:
Just reverted the change in my latest commit!